server/validation/validateAddProject.js

```diff
@@ -0,0 +1,13 @@
+const yup = require('yup');
+
+const addProjectSchema = yup.object({
+  name: yup.string().required(),
+  description: yup.string().required(),
+  imgUrl: yup.string().url().required(),
+  githubLink: yup.string().url().required(),
+  websiteLink: yup.string().url().required(),
+  projectType: yup.string().required(),
```

Comment:
There are only two types of projects so you should prevent the user from entering others
```suggestion
  projectType: yup.mixed().oneOf(['Internal project', 'Remotely project']),
```

Reply:
I think we consider that in the future may be another type of project will make, so don't wont to be limited.
What's you advice, Change it or not?